templates/tfengine/components/cicd/configs/run.sh

```diff
@@ -20,6 +20,11 @@ ACTIONS=()
 DIRS=()
 ROOT="."
 
+LATEST_GCLOUD_VERSION=356.0.0
+
+apk add --no-cache curl
```

Comment:
Do we need `curl` if we are installing the latest gcloud here instead of relying on the CFT module to curl and install?

Reply:
I got this error when omitting the curl installation:

> Step #2 - "Apply": Error: Error running command 'curl -sL -o .terraform/modules/gcloud_debug/cache/60ca7ae6/google-cloud-sdk.tar.gz https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-356.0.0-linux-x86_64.tar.gz': exit status 127. Output: /bin/sh: curl: not found

Reply:
Seems like it still uses curl under the hood